data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -12,6 +12,17 @@ function wait_for_api {
   done
 }
 
+# This is required since the progress service (https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/report-progress.sh#L22-L33),
```

Comment:
Line links to a branch are unstable, use the "Copy permalink" feature in GitHub instead

Reply:
Thanks, copied the wrong one